File: app/src/main/java/com/example/tutor/ui/category/CategoryViewModel.java
package com.example.tutor.ui.category;

public class CategoryViewModel {
    private String documentID;
    private String name;

    public CategoryViewModel(String documentID, String name) {
        this.documentID = documentID;
        this.name = name;
    }

    public String getDocumentID() {
        return documentID;
    }

    public void setDocumentID(String documentID) {
        this.documentID = documentID;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: app/src/main/java/com/example/tutor/home/fragments/category/CategoryViewModel.kt
package com.example.tutor.home.fragments.category

class CategoryViewModel() {
    private lateinit var documentId: String
    private lateinit var name: String
    private var numberOfQuizzes: Int? = null

    constructor(documentId: String, name: String, numberOfQuizzes: Int) : this() {
        this.documentId = documentId
        this.name = name
        this.numberOfQuizzes = numberOfQuizzes
    }

    fun getName(): String {
        return name
    }

    fun getNumberOfQuizzes(): Int? {
        return numberOfQuizzes
    }
}
